apps/meteor/tests/e2e/e2e-encryption.spec.ts

```diff
@@ -212,4 +212,32 @@ test.describe.serial('e2e-encryption', () => {
 		await expect(poHomeChannel.content.lastUserMessageBody).toHaveText('hello world');
 		await expect(poHomeChannel.content.lastUserMessage.locator('.rcx-icon--name-key')).toBeVisible();
 	});
+
+	test('expect force logout on e2e keys reset', async ({ page, browser }) => {
+		const poAccountProfile = new AccountProfile(page);
+		// creating another logged in client, to check force logout
+		const { page: anotherClientPage } = await createAuxContext(browser, Users.admin);
+
+		await page.goto('/account/security');
+
+		await poAccountProfile.securityE2EEncryptionSection.click();
+		await poAccountProfile.securityE2EEncryptionResetKeyButton.click();
+
+		await page.locator('role=button[name="Login"]').waitFor();
+		await anotherClientPage.locator('role=button[name="Login"]').waitFor();
+
+		await expect(page).toHaveURL('/home');
+		await expect(anotherClientPage).toHaveURL('/home');
+
+		await expect(page.locator('role=button[name="Login"]')).toBeVisible();
+		await expect(anotherClientPage.locator('role=button[name="Login"]')).toBeVisible();
+
+		await expect(page.locator('role=banner')).toContainText('Your session was ended on this device, please log in again to continue.');
+		await expect(anotherClientPage.locator('role=banner')).toContainText('Your session was ended on this device, please log in again to continue.');
+
+		await anotherClientPage.close();
+
+		// inject initial data, so that tokens are restored after forced logout
+		await injectInitialData();
```

Comment:
Should be on an `after` call (afterall maybe)

Reply:
Makes sense.